torch/utils/hipify/hipify_python.py

```diff
@@ -989,7 +989,7 @@ def repl(m):
             hipify_result.status = "[ok]"
             hipify_result.current_state = CurrentState.DONE
             return hipify_result
-        except PermissionError as e:
+        except (PermissionError, OSError) as e:
             print(f'{bcolors.WARNING}Failed to save {fout_path} with "{e.strerror}", leaving {fin_path} unchanged.{bcolors.ENDC}',
```

Comment:
this should be a logger call, not a print statement...

Reply:
I checked the `hipify_python.py`, and it seems like everything is done by `print` and there is no usage of logger handlers in the file here...

Plus, this PR doesn't touch this directly, so it's kind of out of the scope, and this can be refactored altogether in a separate PR.